src/scene/shader-lib/chunks/lit/frag/shadowPCSS.js

```diff
@@ -0,0 +1,185 @@
+export default /* glsl */`
+
+
+#define PCSS_SAMPLE_COUNT 16
```

Comment:
considering the number of samples is a hardcoded define, should we specify vogel disk and sphere offsets as inline constants to avoid their evaluation?

Reply:
I see now .. it depends on screen pixel noise and so it can only be partially precomputed. Perhaps even that is worth it?

Reply:
Sure but then we lose the ability to change the number of samples.

Reply:
in that case, we should have it specified in JS code, which would generate a define for shader + appropriate look up tables. But avoid doing this per pixel.

Reply:
yeah that's what I thought, but I'll leave the decision to you if it's worth the gain here.

Reply:
Yeah, did a partial precalculation which saves us a few sqrt.